apps/s_server.c

```diff
@@ -147,13 +147,14 @@ static int dtlslisten = 0;
 static char *psk_identity = "Client_identity";
 char *psk_key = NULL;           /* by default PSK is not used */
 
-static unsigned int psk_server_cb(SSL *ssl, const char *identity,
+static unsigned int psk_server_cb(SSL *s, const char *identity,
```

Comment:
yes, there are a few places where habits kicked in and I did some other code-cleanup.  Working on backing all of those out now.

perhaps you're right about the unused thing.  i'll think about it.